implementation/requests/http.go

```diff
@@ -191,7 +191,7 @@ func checkSubdomains(apiAddress *url.URL, requestedURL *url.URL) error {
 
 func validateURL(connection map[string]string, requestedURL *url.URL, plugin types.Plugin) error {
 	// try to get the request url from the connection
-	apiAddressString, ok := connection[consts.RequestUrlKey]
+	apiAddressString, ok := connection[consts.ApiAddressKey]
 	if !ok {
 		apiAddressString, ok = connection[consts.ApiAddressKey]
 		if !ok {
```

Comment:
```suggestion
	apiAddressString, ok := connection[consts.ApiAddressKey]
	if !ok {
```
no longer need to double check, only one key